extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -289,14 +301,18 @@ export class AmpLiveList extends AMP.BaseElement {
    *
    * @param {!Element} parent
    * @param {!Array<!Element>} orphans
+   * @return {number}
```

Comment:
Document: New item count.

Please also add elsewhere, because it isn't always clear what a number is.